build.gradle

```diff
@@ -94,6 +94,8 @@ dependencies {
   }
   api "commons-io:commons-io:2.11.0"
 
+  api 'com.networknt:json-schema-validator:1.0.78'
```

Comment:
Looks to be the best possible choice among available validators. I used https://github.com/everit-org/json-schema before but there are strange deprecation events that happened this week

Reply:
Agree this looks like the best one.